x11/xorg-libxcb/Portfile

```diff
@@ -4,6 +4,7 @@ PortSystem      1.0
 
 name            xorg-libxcb
 version         1.13.1
+revision        1
```

Comment:
Does this change alter the resulting binaries? If not, then do we need to bump the version number?

Reply:
Some Python scripts (xcbgen) are moved from `$prefix/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages` to `$prefix/opt/local/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages`. It might not be a big change for a bump.